cogs/dejv.py

```diff
@@ -16,14 +12,13 @@ def __init__(self, bot: commands.Bot):
 
     @commands.Cog.listener()
     async def on_message(self, message: Message):
-
-        nalezeno = False
-
         """
         When message with specific content from activation_strings
         is detected the bot sends response.
         """
 
+        nalezeno = False
```

Comment:
OMG. Please STOP use fucking czech in the code. STOP!!!

Reply:
@CptToothpick good job